### PR TITLE
turn in lession 14 and 15

### PR DESCRIPTION
**Pull request by 29jinghong**

### CODE REVIEW

**Comment by wanggang1** [comment]
Looks good.  You need to change my role or assign me rights so I can comment on your changes.  We can discuss in the meeting

**Comment by alastor1729** [comment]
same as Gang's PR message, @29jinghong

**Comment by alastor1729** [comment]
nevermind;
@wanggang1, I can comment on JingHong's PR now.

**Comment by wanggang1** [comment]
@alastor1729  I meant commenting in the code, not here.   Can you add comments in his code?  I can now once I accepted the invitation

File: 14/14.1 [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import urllib.request, urllib.parse, urllib.error
```
Comment (alastor1729) [inline 14/14.1:1]: @wanggang1, I can comment on his code

Comment (alastor1729) [reply 14/14.1:1]: Can you not click on the + icon, next to the code?

** let me know if you want me to send a screenshot or something

**Pull request merged**